unit-tests-java8/src/test/java/org/eclipse/collections/test/stack/mutable/UnmodifiableMutableStackTestCase.java

```diff
@@ -24,18 +25,18 @@ default void Iterable_remove()
     }
 
     @Override
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     default void MutableStack_pop()
     {
         MutableStack<Integer> mutableStack = this.newWith(5, 1, 4, 2, 3);
-        mutableStack.pop();
+        Assert.assertThrows(UnsupportedOperationException.class, () -> mutableStack.pop());
     }
 
     @Override
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     default void MutableStack_pop_throws()
     {
         MutableStack<Integer> mutableStack = this.newWith();
-        mutableStack.pop();
+        Assert.assertThrows(UnsupportedOperationException.class, () -> mutableStack.pop());
```

Comment:
The other usages are static imports. I think these are the only usages with non-static imports. What's the distinction here?